fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -2408,8 +2424,8 @@ public int getAnalyzeTimeoutS() {
         return analyzeTimeoutS;
     }
 
-    public void setEnableTwoPhaseReadOpt(boolean enable) {
-        enableTwoPhaseReadOpt = enable;
+    public void disableTwoPhaseReadOpt() {
```

Comment:
we should use `enable` as function or variable name, disable is hard to understand